### PR TITLE
fix: cache and database use specify storageclass

### PR DESCRIPTION
**Pull request by chlins**
Fix: https://github.com/goharbor/harbor-operator/issues/330

Signed-off-by: chlins <chlins.zhang@gmail.com>

### CODE REVIEW

**Review by Ghostbaby — APPROVED** [review]
LGTM

File: pkg/cluster/controllers/cache/resource_manager.go [hunk 1]
```diff
@@ -223,3 +223,12 @@ func (rm *redisResourceManager) GetStorageSize() string {
...
 	return rm.cluster.Spec.InClusterCache.RedisSpec.Server.Storage
 }
+
+// GetStorageClass gets the storage class name.
+func (rm *redisResourceManager) GetStorageClass() string {
+	if rm.cluster.Spec.InClusterCache.RedisSpec != nil && rm.cluster.Spec.InClusterCache.RedisSpec.Server != nil {
+		return rm.cluster.Spec.InClusterCache.RedisSpec.Server.StorageClassName
+	}
+
+	return ""
```
Comment (steven-zou) [inline pkg/cluster/controllers/cache/resource_manager.go:233]: no default class?

**Review by steven-zou — COMMENTED** [review]
see my question?

**Review by steven-zou — COMMENTED** [review]
approved

**Review by steven-zou — APPROVED** [review]
APPROVED

**Comment by steven-zou** [comment]
@bitsf 

It seems the ci is getting non-stable now.

**Pull request merged**